runtime/src/snapshot_package.rs

```diff
@@ -39,6 +39,8 @@ pub struct AccountsPackage {
     pub slot_deltas: Vec<BankSlotDelta>,
     pub snapshot_links: TempDir,
     pub snapshot_storages: SnapshotStorages,
+    /// necessary for calculating hash in accounts hash verifier
+    pub all_snapshot_storages: Option<SnapshotStorages>,
```

Comment:
I feel that `snapshot_packages` could just have all the snapshot storages, and not need to have two fields for the snapshot storages. Then, when creating the SnapshotPackage, for incremental snapshots, filter out storages older than the full snapshot slot. Can that be done?